### PR TITLE
added to accept environment variable and yaml file for configuration

### PR DESCRIPTION
**Pull request by kjeom**
## Proposed changes

- Describe your changes to communicate to the maintainers why we should accept this pull request.
- If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- 

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: api/debug/flags.go [hunk 1]
```diff
@@ -31,74 +31,95 @@ import (
 	"github.com/klaytn/klaytn/log"
 	"github.com/klaytn/klaytn/log/term"
 	colorable "github.com/mattn/go-colorable"
+	"github.com/urfave/cli/altsrc"
```
Comment (odg0318) [inline api/debug/flags.go]: "gopkg.in/urfave/cli.v1/altsrc" 를 쓰는게 좋아보입니다.

File: cmd/utils/nodecmd/nodeflags.go [hunk 2]
```diff
@@ -23,351 +23,352 @@ package nodecmd
...
-	utils.IdentityFlag,
-	utils.UnlockedAccountFlag,
-	utils.PasswordFileFlag,
-	utils.DbTypeFlag,
+	altsrc.NewStringFlag(utils.BootnodesFlag),
+	altsrc.NewStringFlag(utils.IdentityFlag),
+	altsrc.NewStringFlag(utils.UnlockedAccountFlag),
+	altsrc.NewStringFlag(utils.PasswordFileFlag),
+	altsrc.NewStringFlag(utils.DbTypeFlag),
 	utils.DataDirFlag,
```
Comment (odg0318) [inline cmd/utils/nodecmd/nodeflags.go]: 이건 변경이 안되나요?

File: cmd/utils/nodecmd/nodeflags.go [hunk 3]
```diff
@@ -23,351 +23,352 @@ package nodecmd
...
-	utils.OverwriteGenesisFlag,
-	utils.StartBlockNumberFlag,
+	altsrc.NewBoolFlag(utils.OverwriteGenesisFlag),
+	altsrc.NewUint64Flag(utils.StartBlockNumberFlag),
 	utils.KeyStoreDirFlag,
```
Comment (odg0318) [inline cmd/utils/nodecmd/nodeflags.go]: 작업 필요해보입니다.

File: cmd/utils/nodecmd/nodeflags.go [hunk 4]
```diff
@@ -23,351 +23,352 @@ package nodecmd
...
+	altsrc.NewDurationFlag(utils.TxPoolJournalIntervalFlag),
+	altsrc.NewUint64Flag(utils.TxPoolPriceLimitFlag),
+	altsrc.NewUint64Flag(utils.TxPoolPriceBumpFlag),
+	altsrc.NewUint64Flag(utils.TxPoolExecSlotsAccountFlag),
+	altsrc.NewUint64Flag(utils.TxPoolExecSlotsAllFlag),
+	altsrc.NewUint64Flag(utils.TxPoolNonExecSlotsAccountFlag),
+	altsrc.NewUint64Flag(utils.TxPoolNonExecSlotsAllFlag),
+	altsrc.NewDurationFlag(utils.TxPoolLifetimeFlag),
+	altsrc.NewBoolFlag(utils.TxPoolKeepLocalsFlag),
 	utils.SyncModeFlag,
```
Comment (odg0318) [inline cmd/utils/nodecmd/nodeflags.go]: 작업 필요해보입니다.

File: cmd/utils/nodecmd/nodeflags.go [hunk 5]
```diff
@@ -23,351 +23,352 @@ package nodecmd
...
+	altsrc.NewIntFlag(utils.GRPCPortFlag),
+	altsrc.NewIntFlag(utils.RPCConcurrencyLimit),
+	altsrc.NewStringFlag(utils.WSApiFlag),
+	altsrc.NewStringFlag(utils.WSAllowedOriginsFlag),
+	altsrc.NewIntFlag(utils.WSMaxSubscriptionPerConn),
+	altsrc.NewInt64Flag(utils.WSReadDeadLine),
+	altsrc.NewInt64Flag(utils.WSWriteDeadLine),
+	altsrc.NewIntFlag(utils.WSMaxConnections),
+	altsrc.NewBoolFlag(utils.IPCDisabledFlag),
 	utils.IPCPathFlag,
```
Comment (odg0318) [inline cmd/utils/nodecmd/nodeflags.go]: 작업 필요해보입니다.

File: cmd/utils/nodecmd/nodeflags.go [hunk 6]
```diff
@@ -23,351 +23,352 @@ package nodecmd
...
+	altsrc.NewUint64Flag(utils.MaxBlockDiffFlag),
+	altsrc.NewUint64Flag(utils.TxResendIntervalFlag),
+	altsrc.NewIntFlag(utils.TxResendCountFlag),
+	altsrc.NewBoolFlag(utils.TxResendUseLegacyFlag),
 }
 
 var DBMigrationFlags = []cli.Flag{
-	utils.DstDbTypeFlag,
+	altsrc.NewStringFlag(utils.DstDbTypeFlag),
 	utils.DstDataDirFlag,
```
Comment (odg0318) [inline cmd/utils/nodecmd/nodeflags.go]: 작업 필요해보입니다.

File: cmd/utils/nodecmd/defaultcmd.go [hunk 7]
```diff
@@ -35,6 +35,7 @@ import (
 	metricutils "github.com/klaytn/klaytn/metrics/utils"
 	"github.com/klaytn/klaytn/node"
 	"github.com/klaytn/klaytn/node/cn"
+	"github.com/urfave/cli/altsrc"
```
Comment (odg0318) [inline cmd/utils/nodecmd/defaultcmd.go]: "gopkg.in/urfave/cli.v1/altsrc" 를 쓰는게 좋아보입니다.

Comment (kjeom) [reply cmd/utils/nodecmd/defaultcmd.go]: 모든파일 하나의 패키지로 통일하니 잘 작동하네요 :)

File: cmd/utils/nodecmd/nodeflags.go (hunk 6 above)
Comment (kjeom) [reply cmd/utils/nodecmd/nodeflags.go]: directoryFlag, textMarshalFlag yaml config추가되었어요

**Pull request closed without merging**